prow/config/config.go

```diff
@@ -2484,7 +2495,6 @@ func parseProwConfig(c *Config) error {
 
 		if templates.BodyTemplate != "" {
 			bodyTemplate, err := template.New("CommitBody").Parse(templates.BodyTemplate)
-
```

Comment:
Let's not introduce diffs like these that are not related to the code that's being changed.

Reply:
Removed 👍